app/components/search/Header/Header.module.scss

```diff
@@ -0,0 +1,78 @@
+@import "~styles/utils/_helpers.scss";
```

Comment:
With the exception of the qr* classes, this styles in this file have been migrated  from: `app/components/search/SearchAndServiceResultsPage.module.scss` below